news/2020-03-13-featured-artist-beatmap-updates-from-the-mappers-guild.md

```diff
@@ -0,0 +1,439 @@
+---
+layout: post
+title: Featured Artist Beatmap Updates from the Mappers' Guild
+date: 2020-03-13 00:00:00 +0000
+---
+
+There's a ton of new beatmaps, completed quests, and medals from the Mappers' Guild celebrating osu!'s featured artists!
+
+![](/wiki/shared/news/banners/mappersguild.png)
+
+Since the [previous update](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) a lot has been happening in the Guild, with over **20** quests completed, over **100** beatmaps ranked, and **16** new medals for you to earn, all tied to the rapidly growing Featured Artist library!
+
+For those who don't know yet, the Mappers' Guild is a way for mappers with three or more ranked beatmaps to earn rewards for creating beatmaps of songs from the Featured Artist library. If you're interested in the Guild, be sure to look at the [Mappers' Guild website](https://mappersguild.com/) for more information.
+
+Recently there has also been some new developments in the Mappers' Guild system! The first is that now anyone can view the website and keep up with what Mappers' Guild members are making, so be sure to check it out if you are curious about what the mappers are working on. Next, the Guild now has solo quests available for members to take on by themselves, giving mappers the option to work alone if that's what they prefer. Lastly, with the return of the [Monthly Beatmapping Contests](https://osu.ppy.sh/help/wiki/Contests/Monthly_Beatmapping_Contest), guild members who participate can also earn points in the Mappers' Guild for doing so, which you can read more about on the linked wiki page.
```

Comment:
"guild" isn't capitalized after the MBC wiki link, whereas everywhere else it is
i think it'd make more sense to leave it uncapitalized when it's just "guild" alone and capitalize when it's the title "Mappers' Guild"